scripts/cli/cmd/erc721.js

```diff
@@ -0,0 +1,75 @@
+const ethers = require('ethers');
+const constants = require('../constants');
+
+const { Command } = require('commander');
+const { setupParentArgs } = require("./utils")
+
+const BridgeContract = require("../../../build/contracts/Bridge.json");
+const ERC721Contract = require("../../../build/contracts/ERC721Mintable.json");
+
+const mintCmd = new Command("mint")
```

Comment:
I think the `mint` command should have a `--id` argument like the `transfer` command does - if not, you can only mint a NFT with ID 1